### PR TITLE
Rename entpanels posts; add opacity post

### PR DESCRIPTION
**Pull request by geight**
Closes https://github.com/momentum-mod/docs/issues/214

The GitHub diff is acting like `2020-04-30-mom_entpanels_enable_names.md` was deleted entirely and `2020-04-30-mom_hud_ghost_entpanels_enable.md` was created instead, but all of the files were renamed with the updated command name(s):

updated_entpanels_posts

> I opted not to rename the original files with today's date, but that can be done if preferred.

### CODE REVIEW

File: _posts/convars/2020-04-30-mom_hud_ghost_entpanels_enable_avatars.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: mom_entpanels_enable_avatars
+title: mom_hud_ghost_entpanels_enable_avatars
```
Comment (braem) [inline _posts/convars/2020-04-30-mom_hud_ghost_entpanels_enable_avatars.md:2]: This is `mom_hud_ghost_entpanels_avatars_enable`. Forgot to include that in the issue sorry!

Comment (geight) [reply _posts/convars/2020-04-30-mom_hud_ghost_entpanels_enable_avatars.md]: Resolved in [`2808c19` (#230)](https://github.com/momentum-mod/docs/pull/230/commits/2808c19000f132ec30c7ae65c39ec71f42462f11)

File: _posts/convars/2020-04-30-mom_hud_ghost_entpanels_enable_names.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: mom_entpanels_enable
+title: mom_hud_ghost_entpanels_enable_names
```
[the remark this answers is not in the document]
Comment (geight) [reply _posts/convars/2020-04-30-mom_hud_ghost_entpanels_enable_names.md]: Resolved in [`2808c19` (#230)](https://github.com/momentum-mod/docs/pull/230/commits/2808c19000f132ec30c7ae65c39ec71f42462f11)

**Review by braem — APPROVED** [review]

**Comment by braem** [comment]
Do you have a key? DM me or goc if you want one!

**Pull request merged**